server_admin/topics/users/attributes.adoc

```diff
@@ -1,11 +1,18 @@
+[configuring-user-attributes]
+= Configuring user attributes
 
-=== User Attributes
+In addition to adding basic user metadata, such as names and email, you can also store arbitrary user attributes.  
 
-Beyond basic user metadata like name and email, you can store arbitrary user attributes.  Choose a user to manage
-then click on the `Attributes` tab.
+.Prerequisite
+* You are logged into the correct realm.
```

Comment:
.Prerequisite
You are logged into the realm with the user that you intend to modify.